hasura/metadata/databases/staging/tables/public_pending_gift_private.yaml

```diff
@@ -0,0 +1,23 @@
+table:
+  name: pending_gift_private
+  schema: public
+select_permissions:
+- permission:
+    columns:
+    - recipient_id
+    - sender_id
+    - note
+    filter: {}
+  role: superadmin
+- permission:
+    columns:
+    - recipient_id
+    - sender_id
+    - note
+    filter:
+      _or:
+      - recipient_id:
+          _in: X-Hasura-Circle-User-Ids
+      - sender_id:
+          _in: X-Hasura-Circle-User-Ids
+  role: user
```

Comment:
Lets say we want to allow circle admins to see `recipient_id` and `sender_id` but not the `note`. That would be a use case where we want to setup a separate `circle-admin` role (vs `id:  _in: X-Hasura-Admin-Circle-Ids`).

Reply:
we want to only allow the recipient / sender of the note to be able to view the note. circle admins are also restricted to this same condition. Is there a different way to accomplish this compared to view ?